### PR TITLE
EDX Translations Updated for Arabic Language

### PR DESCRIPTION
**Pull request by asadmanzoor93**
**Description:**  This PR adds missing translations for Arabic language support in Edly Edx Themes

**JIRA:** 
https://edlyio.atlassian.net/browse/EDLY-1604

**Merge checklist:**

- [ ] All reviewers approved
- [ ] Commits are (reasonably) squashed

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

**Note:** Please add screenshots for any viewable change.

### CODE REVIEW

File: cms/static/js/i18n/ar/djangojs.js [hunk 1]
```diff
@@ -543,7 +543,7 @@
     "Create an Account": "\u0625\u0646\u0634\u0627\u0621 \u062d\u0633\u0627\u0628", 
     "Create an Account.": "\u0625\u0646\u0634\u0627\u0621 \u062d\u0633\u0627\u0628.", 
     "Create an account using": "\u0623\u0646\u0634\u0626 \u062d\u0633\u0627\u0628\u0627\u064b \u0628\u0627\u0633\u062a\u062e\u062f\u0627\u0645", 
-    "Create an {platform} account": "\u0625\u0646\u0634\u0627\u0621 \u062d\u0633\u0627\u0628 {platform}",
```
Comment (zubair-arbi) [inline cms/static/js/i18n/ar/djangojs.js:546]: I think this was correct. We should never translate placeholders like `{platform}`.

File: cms/static/js/i18n/ar/djangojs.js [hunk 2]
```diff
@@ -269,7 +269,7 @@
     "Already a course team member": "\u0639\u0636\u0648 \u0645\u0633\u0628\u0642 \u0641\u064a \u0641\u0631\u064a\u0642 \u0627\u0644\u0645\u0633\u0627\u0642", 
     "Already a library team member": "\u0639\u0636\u0648 \u0645\u0633\u0628\u064e\u0642 \u0641\u064a \u0641\u0631\u064a\u0642 \u0627\u0644\u0645\u0643\u062a\u0628\u0629", 
     "Already a member": "\u0639\u0636\u0648 \u0645\u0633\u062c\u0651\u064e\u0644 \u0645\u0646 \u0642\u0628\u0644", 
-    "Already have an account?": "\u0647\u0644 \u0644\u062f\u064a\u0643 \u062d\u0633\u0627\u0628 \u0645\u0633\u0628\u0642\u0627\u064b\u061f", 
+    "Already have an account?": "\u0647\u0644 \u0644\u062f\u064a\u0643 \u062d\u0633\u0627\u0628 {platformName} \u0645\u0633\u0628\u0642\u0627\u064b\u061f",
```
Comment (zubair-arbi) [inline cms/static/js/i18n/ar/djangojs.js:272]: Why we needed to add placeholder `{platformName}` in translated string?

File: cms/static/js/i18n/ar/djangojs.js [hunk 3]
```diff
@@ -871,7 +870,7 @@
...
-    "If you don't verify your identity now, you can still explore your course from your dashboard. You will receive periodic reminders from {platformName} to verify your identity.": "\u0625\u0630\u0627 \u0644\u0646 \u062a\u0642\u0645 \u0628\u0627\u0644\u062a\u062d\u0642\u0642 \u0645\u0646 \u0647\u0648\u064a\u062a\u0643 \u0627\u0644\u0622\u0646\u060c \u0644\u0627 \u064a\u0632\u0627\u0644 \u0628\u0625\u0645\u0643\u0627\u0646\u0643 \u0627\u0633\u062a\u0643\u0634\u0627\u0641 \u0627\u0644\u062f\u0648\u0631\u0629 \u0645\u0646 \u062e\u0644\u0627\u0644 \u0644\u0648\u062d\u0629 \u0627\u0644\u0645\u0639\u0644\u0648\u0645\u0627\u062a \u0627\u0644\u062e\u0627\u0635\u0629 \u0628\u0643. \u0633\u0648\u0641 \u062a\u062a\u0644\u0642\u0649 \u0631\u0633\u0627\u0626\u0644 \u062a\u0630\u0643\u064a\u0631 \u062f\u0648\u0631\u064a\u0629 \u0645\u0646 {platformName} \u0644\u0644\u062a\u062d\u0642\u0642 \u0645\u0646 \u0647\u0648\u064a\u062a\u0643.",
```
Comment (zubair-arbi) [inline cms/static/js/i18n/ar/djangojs.js:874]: This string hasn't changed. Why we need to update its translation?

File: conf/locale/ar/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -5335,14 +5335,12 @@ msgstr ""
 #: themes/st-lutherx/lms/templates/student/edx_ace/passwordreset/email/body.html
 #: themes/st-normanx/lms/templates/student/edx_ace/passwordreset/email/body.html
 msgid "Change my Password"
-msgstr ""
-"تغيير كلمة المرور الخاصة بي"
+msgstr "تغيير كلمة المرور الخاصة بي"
```
Comment (zubair-arbi) [inline conf/locale/ar/LC_MESSAGES/django.po:5338]: revert this. Its the same translation as before.

File: conf/locale/ar/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -5335,14 +5335,12 @@ msgstr ""
...
 
 #: common/templates/student/edx_ace/passwordreset/email/subject.txt
 #, python-format
 msgid "Password reset on %(platform_name)s"
-msgstr ""
-"إعادة تعيين كلمة المرور قيد التشغيل %(platform_name)s"
+msgstr "إعادة تعيين كلمة المرور قيد التشغيل %(platform_name)s"
```
Comment (zubair-arbi) [inline conf/locale/ar/LC_MESSAGES/django.po:5343]: revert this. Its the same translation as before.

File: cms/static/js/i18n/ar/djangojs.js (hunk 2 above)
Comment (asadmanzoor93) [reply cms/static/js/i18n/ar/djangojs.js:272]: Edx translations auto-update did this.. I will review..

File: cms/static/js/i18n/ar/djangojs.js (hunk 3 above)
Comment (asadmanzoor93) [reply cms/static/js/i18n/ar/djangojs.js:874]: Translation reverted

File: conf/locale/ar/LC_MESSAGES/django.po (hunk 5 above)
Comment (asadmanzoor93) [reply conf/locale/ar/LC_MESSAGES/django.po]: Done

File: conf/locale/ar/LC_MESSAGES/django.po (hunk 4 above)
Comment (asadmanzoor93) [reply conf/locale/ar/LC_MESSAGES/django.po]: Done

File: cms/static/js/i18n/ar/djangojs.js (hunk 1 above)
Comment (asadmanzoor93) [reply cms/static/js/i18n/ar/djangojs.js]: Translation reverted.

**Pull request merged**